.github/workflows/branch-pr-release.yaml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+    - created
```

Comment:
```suggestion
    - published
```

Reply:
Basing this on https://code.visualstudio.com/api/working-with-extensions/continuous-integration#github-actions What are the benefits of changing this?